packages/medusa-js/src/resources/admin/auth.ts

```diff
@@ -41,6 +42,27 @@ class AdminAuthResource extends BaseResource {
     const path = `/admin/auth`
     return this.client.request("POST", path, payload, {}, customHeaders)
   }
+
+  /**
+   * @description Retrieves a new JWT access token
+   * @param {AdminPostAuthReq} payload
+   * @param customHeaders
+   * @return {ResponsePromise<AdminBearerAuthRes>}
+   */
+  getToken(
+    payload: AdminPostAuthReq,
+    customHeaders: Record<string, any> = {}
+  ): ResponsePromise<AdminBearerAuthRes> {
+    const path = `/admin/auth/token`
+    return this.client.request("POST", path, payload, {}, customHeaders)
+      .then((res) => {
+        if (res.data?.access_token) {
```

Comment:
todo: here if res.data.access_token is not present then there is an issue somewhere. I think this check is redundant should we remove it?

Reply:
fixed problems. Now works and has no redundant checks.